src/stories/_wrap.py

```diff
@@ -1,11 +1,15 @@
+from .exceptions import StoryDefinitionError
 from ._contract import combine_contract, make_contract, maybe_extend_downstream_argsets
 from ._failures import combine_failures, make_exec_protocol, maybe_disable_null_protocol
 from ._marker import BeginningOfStory, EndOfStory
 from ._mounted import MountedStory
+from ._exec import get_executor
 
 
 def wrap_story(arguments, collected, cls_name, story_name, obj, spec, failures):
 
+    executor = None
+
```

Comment:
If collected is an empty list, it will be TypeError in the `__call__` and `run` methods of the `MountedStory`. I think we should deny empty stories in the `_collector` module. Do empty stories make any sense?